packages/in_app_purchase/in_app_purchase_storekit/darwin/Classes/FIAObjectTranslator.h

```diff
@@ -56,7 +57,18 @@ NS_ASSUME_NONNULL_BEGIN
                                                   withError:(NSString *_Nullable *_Nullable)error
     API_AVAILABLE(ios(12.2));
 
++ (nullable SKPaymentTransactionMessage *)convertTransactionToPigeon:
+    (SKPaymentTransaction *)transaction;
```

Comment:
nit: This parameter should be marked `nullable` since it handles null values, right? Same for `convertStorefrontToPigeon:`, `convertPaymentDiscountToPigeon:`, and `convertPaymentToPigeon:`.
```suggestion
    (nullable SKPaymentTransaction *)transaction;
```